src/umpire/resource/MemoryResourceTypes.hpp

```diff
@@ -30,7 +30,8 @@ enum MemoryResourceType {
   Pinned,
   Constant,
   File,
-  Unknown
+  Unknown,
```

Comment:
Personallhy, I kind of prefer to either make `Unknown` be the last entry or the first entry in the enumerations.  I really can't tell you why though :-).

Reply:
Nah, I get it. I feel that is kinda better too lol